Validation/RecoVertex/src/AnotherPrimaryVertexAnalyzer.cc

```diff
@@ -47,7 +47,7 @@
 // class decleration
 //
 
-class AnotherPrimaryVertexAnalyzer : public edm::EDAnalyzer {
+class AnotherPrimaryVertexAnalyzer : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
```

Comment:
The `VertexHistogramMaker` helper appears to use `TFileService`, so the constructor of this class would need
```cpp
  usesResource(TFileService::kSharedResource);
```

Reply:
done.